### DISCUSSION TITLE
Welcome to vanillaSelectBox Discussions!

### DISCUSSION THREAD

**Discussion by PhilippeMarcMeyer** [PhilippeMarcMeyer/vanillaSelectBox#22] [General]
## 👋 Welcome to this space of discussion :

 Feel free to ask and discuss, suggest :-)

**Comment by PhilippeMarcMeyer** [comment]
You can talk about other projets or needs or ideas of yours

**Comment by PhilippeMarcMeyer** [comment]
Check my todo list at the end of the readme file !

**Comment by piburglar** [comment]
So, what do you do when your lists contain commas? Mainly when setting the values 
When I do example.setValue(), either with an array of values which contain commas, or a list of values containing commas, it doesnt work.
